### PR TITLE
Added a new app spec

### PR DESCRIPTION
**Pull request by AbdulKhadhar**
Added a new app specification

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # HAPPY BIRTHDAY
 _Flutter app with a bday wish_ 
+_with a bday cake image_
```
Comment (RamsiyaKB) [inline README.md]: change bday as birthday

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # HAPPY BIRTHDAY
 _Flutter app with a bday wish_
```
Comment (RamsiyaKB) [inline README.md]: expand bday

**Review by RamsiyaKB — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AbdulKhadhar) [reply README.md]: okey

File: README.md (hunk 2 above)
Comment (AbdulKhadhar) [reply README.md]: okey deal

**Review by RamsiyaKB — APPROVED** [review]

**Pull request merged**